zk.js/src/transaction/fetch-recent-transactions.ts

```diff
@@ -250,7 +402,35 @@ async function enrichParsedTransactionEvents(
     Buffer.from(encryptedUtxos),
     leaves.length,
   );
-  transactions.push({
+  // transactions.push({
+  //   IDs,
+  //   merkleTreePublicKey:
+  //     MerkleTreeConfig.getTransactionMerkleTreePda().toBase58(),
+  //   transaction: {
+  //     blockTime: tx.blockTime! * 1000,
+  //     signer: accountKeys[0],
+  //     signature,
+  //     to,
+  //     from,
+  //     //TODO: check if this is the correct type after latest main?
+  //     //@ts-ignore
+  //     toSpl,
+  //     fromSpl,
+  //     verifier: verifier.toBase58(),
+  //     rpcRecipientSol,
+  //     type,
+  //     changeSolAmount: changeSolAmount.toString("hex"),
+  //     publicAmountSol: amountSol.toString("hex"),
+  //     publicAmountSpl: amountSpl.toString("hex"),
+  //     encryptedUtxos: encryptedUtxos,
+  //     leaves,
+  //     nullifiers,
+  //     rpcFee: rpcFee.toString("hex"),
+  //     firstLeafIndex: firstLeafIndex.toString("hex"),
+  //     message: message,
+  //   },
+  // });
+  return {
     IDs,
     merkleTreePublicKey: MERKLE_TREE_SET.toBase58(),
```

Comment:
we're currently using a const for the set, i suspect eventually this will have to be dynamic?